docker/entrypoint-prod.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+mkdir /var/jenkins_home/userContent
+wget https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css -O /var/jenkins_home/userContent/bootstrap.min.css
```

Comment:
Is this one necessary if we're running the release checker on intg Jenkins?

Reply:
Ah good point yeah, I've taken it out.